### ISSUE TITLE
alt+ up/down does nothing

### ISSUE THREAD

**Issue by phazei** [fnagel/jquery-ui#42]
I thought this was reported before, but I didn't see it in 'search issues' so...

It is standard functionality in IE/FF/Opera for ALT+UP/DOWN, on a select menu, to open up the menu and have the currently selected item be highlighted.

Current functionality opens the menu when space is pressed, which is non-standard as space doesn't do anything in other browsers, and when it opens, nothing is selected most of the time.

**Comment by fnagel** [comment]
Please see this issue for discussion about keyboard control:
https://github.com/fnagel/jquery-ui/issues/closed#issue/6

You're right should be implemented. I consider this as a feature.

**Comment by fnagel** [comment]
I implemented this feature. I will do some tests and commit this soon. Please be patient.

**Comment by fnagel** [comment]
Please check out the latest commit and give feedback!

**Comment by phazei** [comment]
All testing done in Chrome v8 and FF

A couple of things to note:
ALT+LEFT/RIGHT should still work as forward/back in the browser.  I noticed this when I tried to go back a page and it only changed the selected item.  Left/Right by themselves should change the option.

When using Alt+UP/DOWN,  the selected item isn't highlighted when it opens up.  It seems to have slightly different behavior depending on the 'style'.  Some of them it has an outline on the selected item with no highlight, others it doesn't.  Also it varies if the last item was selected with the mouse, the arrows, or the keyboard pop-up.

There is also an issue with when an item is selected.  On a regular select box, when it's popped open with the keyboard, the selection changes when you arrow up/down.  This one does that when closed, but when opened with the keyboard, enter needs to be pressed to select it when it should just be selected.  Note that when using the mouse, it should not be auto-selected if only highlighted with the mouse.

The final issue is the onChange trigger.  This varies per browser.  I believe on IE, if you use the arrows to move then every time you hit up/down, the trigger is changed so if there are other objects on the page that are affected by the selection, they change.  OTOH, in other browsers, I don't believe the onChange trigger is called until the select box itself is unfocused _OR_ enter is pressed.  So if I want to flip through options and see what they change, it's a sequence of down, enter, down, enter, etc.  On IE it's just down, down, down.

I don't believe it needs to match functionality perfectly for each quirk of each browser, but it should probably match one of them (not IE).  Either FF or Chrome I suppose.  I don't know if the standard is specific on the trigger events.

**Comment by phazei** [comment]
Oh, also if I open an option with the mouse and highlight something, then click somewhere outside to unfocus, it closes and the option doesn't change.  But if I click the dropdown again, with mouse or keyboard, the last thing the mouse hovered over is highlighted.  That shouldn't be the case, it should be what ever is selected.
